### PR TITLE
Fat Update

### PR DESCRIPTION
**Pull request by woodythenoob**

### CODE REVIEW

**Comment by woodythenoob** [comment]
API:

postOperations
-getLikesForPost (gets the likes)
-getUsersPosts (gets the posts by UID)
-likePost (will like a post and update db)
-unlinkePost (will unlike and update db)

userOperations
-searchUserbyUsername : will take uid and get the username
-searchUsers : will use UID to get inforamoint

Pages:
userPosts - my testing page for the enviroment

File: src/pages/api/postOperations/createMockPosts.js
Comment (dobsonddev) [inline src/pages/api/postOperations/createMockPosts.js:1]: good

File: src/pages/api/postOperations/createPost.js
Comment (dobsonddev) [inline src/pages/api/postOperations/createPost.js:1]: good

File: src/pages/api/postOperations/getLikesForPost.js
Comment (dobsonddev) [inline src/pages/api/postOperations/getLikesForPost.js:1]: good

File: src/pages/api/postOperations/getUserPosts.js
Comment (dobsonddev) [inline src/pages/api/postOperations/getUserPosts.js:1]: good

File: src/pages/api/postOperations/likePost.js
Comment (dobsonddev) [inline src/pages/api/postOperations/likePost.js:1]: good

File: src/pages/api/postOperations/unlikePost.js
Comment (dobsonddev) [inline src/pages/api/postOperations/unlikePost.js:1]: good

File: src/pages/api/userOperations/registrationHandler.js
Comment (dobsonddev) [inline src/pages/api/userOperations/registrationHandler.js:1]: good

File: src/pages/api/userOperations/searchUserByUsername.js
Comment (dobsonddev) [inline src/pages/api/userOperations/searchUserByUsername.js:1]: good

File: src/pages/api/userOperations/searchUsers.js
Comment (dobsonddev) [inline src/pages/api/userOperations/searchUsers.js:1]: good

**Review by dobsonddev — COMMENTED** [review]
i think this should be okay merge (smiley face)

**Review by dobsonddev — APPROVED** [review]
approve

**Pull request merged**